autovalet_lane_detection/README.md

```diff
@@ -12,3 +12,5 @@ roslaunch autovalet_lane_detection lane_detector.launch
 ```
 roslaunch autovalet_lane_detection lane_detector_sim.launch
 ```
+
```

Comment:
Can we make these a little more descriptive? 
Like if we had to run this subsystem individually, what other launch files need to be run along with/ before launching autovalet_lane_detection.

Reply:
added some more info

Reply:
do we need to launch  valet_rtab_sim too?

Reply:
not necessary. Just the lot is enough!